android/src/main/java/nordpol/TagDispatcher.java

```diff
@@ -12,6 +12,8 @@
 import android.os.Bundle;
 
 public class TagDispatcher {
+    private static final int DELAY_PRESENCE = 60000;
```

Comment:
@adriaan-telcred Yes, sadly it is. Please see: https://code.google.com/p/android/issues/detail?id=58773